actions/rootio/v1/Dockerfile

```diff
@@ -35,7 +35,8 @@ RUN ./autogen.sh; ./configure
 RUN make LDFLAGS="--static"
 
 # Build final image
-FROM scratch
+FROM alpine
```

Comment:
This largely defeats the purpose behind building the other fsutils from scratch. Without trying to lookup or justify why that was done or why that continues to be important, we should probably build lvm2 (and any dependent tools) from source as is done above.

Reply:
Building lvm2 statically is not a trivial task. I used the binary from the lvm2-static package and put back scratch for the main image

Reply:
@displague What do you think about this?

Reply:
Thanks for updating how lvm2 is added in a way that doesn't affect the base. I think copying from alpine should work. I don't know if it defeats the principal behind building from source as the others do, if there was one.